### PR TITLE
Feature/layout

### PR DESCRIPTION
**Pull request by nachosoloaga**
## What & Why:
Extracted common components between guest pages (login and signup at first) to create a layout and avoid menu compoennt and social media container re-render on each guest route.

## Tasks:
- [x] Tested in different resolutions
- [ ] Tested with SSR
- [x] Tested on Safari browser
- [x] Design reviewed with design team

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -17,9 +19,15 @@ const App = () => {
       </Helmet>
       <BrowserRouter>
         <Switch>
-          {routes.map((route, index) => (
-            <RouteFromPath key={`route${index}`} {...route} authenticated={authenticated} />
-          ))}
+          <Route path={['/login', '/sign-up']}>
```
Comment (matteo95g) [inline src/components/App.js]: what about taking those values from `src/constants/routesPaths.js`?

File: src/components/routes/GuestSwitch.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { Route, Switch } from 'react-router-dom';
+
+import LoginPage from '../../pages/LoginPage';
+import SignUpPage from '../../pages/SignUpPage';
+
+const GuestSwitch = () => {
+  return (
+    <Switch>
+      <Route path="/login" component={LoginPage} />
+      <Route path="/sign-up" component={SignUpPage} />
```
Comment (matteo95g) [inline src/components/routes/GuestSwitch.js]: same here

File: src/components/routes/PrivateSwitch.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { Switch } from 'react-router-dom';
+import { bool } from 'prop-types';
+
+import PrivateRoute from './PrivateRoute';
+import HomePage from '../../pages/HomePage';
+
+const PrivateSwitch = ({ authenticated }) => {
+  return (
+    <Switch>
+      <PrivateRoute path="/" component={HomePage} authenticated={authenticated} />
```
Comment (matteo95g) [inline src/components/routes/PrivateSwitch.js]: same here :)

File: src/components/App.js (hunk 1 above)
Comment (nachosoloaga) [reply src/components/App.js]: yes! 👌

**Review by JulianPasquale — APPROVED** [review]

**Pull request merged**